db/databaseGenerator.js

```diff
@@ -57,21 +58,25 @@ const stores = [
     storeID: 1,
     storeName: 'Target',
     storeMinimumFreeShipping: 15,
+    logo: 'https://i.imgur.com/KHPqFqS.png',
```

Comment:
I think all all logos\images need to be stored somewhere (such as AWS)

Reply:
I stored them on imgur. AWS may be a better option, which I would revisit later :) Thanks Cal!